directories/src/redux/api/peer.js

```diff
@@ -0,0 +1,26 @@
+import { Observable } from 'rxjs';
+import Peer from 'peerjs';
+
+import {
+  PEER_HOST,
+  PEER_PORT,
+  PEER_PATH,
+  PEER_DEBUG
+} from '../../config/';
+
+export default peer =>
+  Observable.create((observer) => {
+    peer.on('connection', (connection) => {
+      connection.on('data', (data) => {
+        observer.next(data);
+      });
+    });
+  });
+
+export const peerInit = () => {
+	const randomClient =  Math.random().toString(36).replace(/[^a-z]+/g, '').substr(0, 8);
```

Comment:
nit: try to download prettier-js so that it formats your code and automatically aligns it https://github.com/prettier/prettier